docs/deployment/helm.md

```diff
@@ -198,6 +199,7 @@ API's you need to install the prometheus operator in your cluster.
 | `topologyUpdater.podSetFingerprint`           | bool    | true                     | Enables compute and report of pod fingerprint in NRT objects.                                                                                                                                    |
 | `topologyUpdater.kubeletStateDir`             | string  | /var/lib/kubelet         | Specifies kubelet state directory path for watching state and checkpoint files. Empty value disables kubelet state tracking.                                                                     |
 | `topologyUpdater.args`                        | array   | []                       | Additional [command line arguments](../reference/topology-updater-commandline-reference.md) to pass to nfd-topology-updater                                                                      |
+| `topologyUpdater.revisionHistoryLimit`       | integer |                                  | Specify how many old ControllerRevisions for this DaemonSet you want to retain. [revisionHistoryLimit](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)          |
```

Comment:
nit: fix the table formatting (align column with the one(s) above). The tables were just recently formatted/aligned so let's try to keep it that way :)

Reply:
I notice the last table dividers are not aligned, so I assume this is not necessary. I have aligned the others.

Reply:
Thanks @rouke-broersma for the update. The last dividers are not be perfectly aligned and I think that's ok (especially because the text in some of the last columns is super long and it would be impractical to perfectly align the rest with these).